addons/delivery/report/ir_actions_report_templates.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<template id="delivery_report_saleorder_document" inherit_id="sale.report_saleorder_document">
+    <xpath expr="//p[@t-field='doc.note']" position="before">
```

Comment:
Feel free to add a `name` or `id` in the parent template to simplify and strengthen your xpath :)

Reply:
It looks like there are 3 `<p>`  that are all notes in the template. What would an appropriate name be? `doc_note_main`? :thinking:

Reply:
`order_note` maybe? the other notes are payment term/fpos/order lines notes, so I suppose it should be clear enough :thinking:

Reply:
Done. I will also leave the file name the same so it's easier to match it to the file in the sales module. We can always change it later :)